Library/ViewModels/CuratedProjectsViewModel.swift

```diff
@@ -32,6 +33,12 @@ public final class CuratedProjectsViewModel: CuratedProjectsViewModelType, Curat
     self.loadProjects = curatedProjects
 
     self.dismissViewController = self.doneButtonTappedSignal
+
+    self.isLoading = Signal.merge(
+      self.viewDidLoadSignal.mapConst(true),
+      curatedProjects.mapConst(false)
+        .skipRepeats()
```

Comment:
Don't you want the `skipRepeats` to be outside of the `.merge`?

Reply:
Actually why do you need the `.skipRepeats`? 🤔

Reply:
Oh that was because when I used the `scan` function it was emitting more than once. Now that I replaced it with `reduce`, it's no longer necessary. Good catch!